db/migrate/20180614175704_create_books.rb

```diff
@@ -0,0 +1,14 @@
+class CreateBooks < ActiveRecord::Migration[5.2]
+  def change
+    create_table :books do |t|
+      t.string :Genre, null: false
```

Comment:
Take care! In ruby & rails we use snake_case isntead of Camelcase, a lot of things works under that assumption. Feel free to fix it here, remove the schema.rb and rerun the migrations

Reply:
Gotcha!